code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -1,9 +1,3 @@
-/mob/living/carbon/movement_delay()
-	. = ..()
-
-	if(!get_leg_ignore() && legcuffed) //ignore the fact we lack legs
-		. += legcuffed.slowdown
-
```

Comment:
You removed this but the original PR added it.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/87972842/05cc0268-2f09-4e99-a9d7-d2bdd96add33)

https://github.com/tgstation/tgstation/pull/47761/files

I don't get why it's removed.

Reply:
> but the original PR added it.

And it was removed by https://github.com/tgstation/tgstation/pull/47782 . The 2019 port should have removed this code, but didnt. I said in the pr body.

> #1052 missed some things relating to movespeed when porting [tgstation/tgstation#47782](https://github.com/tgstation/tgstation/pull/47782), which I discovered while preparing to refactor movement_delay() for smooth movement and the mobility refactor.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/dda667cf-6c1b-4f4e-b4bb-dcb0ffd99273)